cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AdapterHttpHandler.java

```diff
@@ -276,7 +276,7 @@ public void deleteAdapter(HttpRequest request, HttpResponder responder,
       adapterService.removeAdapter(Id.Namespace.from(namespaceId), adapterName);
       responder.sendStatus(HttpResponseStatus.OK);
     } catch (CannotBeDeletedException e) {
-      responder.sendString(HttpResponseStatus.FORBIDDEN, e.getMessage());
+      responder.sendString(HttpResponseStatus.CONFLICT, e.getMessage());
     } catch (NotFoundException e) {
```

Comment:
@jwang47 let's do this in a separate PR. Tried it, its increasing this PR's scope, not sure I want to do that late in the cycle. Let's do this as part of a bigger cleanup.